[PATCH] redirect the dir of mklibsrc-log.tx

Upstream-Status: Pending

redirect mklibsrc-log.tx to builddir, not /tmp; otherwise mklibsrc-log.tx
maybe unable to be written if other users is building autogen at the same time.

Signed-off-by: Roy Li <rongqing.li@windriver.com>
---
 pkg/libopts/mklibsrc.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pkg/libopts/mklibsrc.sh b/pkg/libopts/mklibsrc.sh
index 416b402..d612fbc 100644
--- a/pkg/libopts/mklibsrc.sh
+++ b/pkg/libopts/mklibsrc.sh
@@ -19,7 +19,7 @@
 ##  with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 set -ex
-exec 2> /tmp/mklibsrc-log.tx
+exec 2> $top_builddir/mklibsrc-log.tx
 
 top_builddir=`cd $top_builddir ; pwd`
 top_srcdir=`cd $top_srcdir ; pwd`
-- 
1.9.1

